src/multiselect.js

```diff
@@ -114,6 +114,21 @@ export class Multiselect {
     }
   }
 
+  /**
+   * Ignore multi-field updates within the given function.
+   * @param {function} func The function to call.
+   */
+  withoutMultiFieldUpdates(func) {
+      const oldGroup = Blockly.Events.getGroup();
+      // Note that this depends on the fact that eventListener_ will ignore events with a group ID.
+      Blockly.Events.setGroup("ignoreMultiFieldUpdates");
+      try {
+          func();
+      } finally {
+        Blockly.Events.setGroup(oldGroup);
+      }
+  }
+
```

Comment:
If the answer to my 2nd checklist question is yes, then I would recommend moving this to `src/index.js` and exporting the function alone instead, since it's a helper function and doesn't correlate to the plugin directly.